Soundify/Soundify/Widgets/TopDetailView/TopDetailView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  TopDetailView.swift
+//  Soundify
+//
+//  Created by Viet Anh on 3/15/20.
+//  Copyright © 2020 VietAnh. All rights reserved.
+//
+
+import UIKit
+
+protocol TopDetailViewDelegate: AnyObject {
+    func leftBarButtonItemClicked()
+    func scrollViewDidScroll(_ scrollView: UIScrollView)
+}
+
+class TopDetailView: UIView {
+    
+    //MARK: - Init
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        commonInit()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    convenience init(frame: CGRect ,with album: Album) {
+        self.init(frame: frame)
+        setUpDetailView(with: album)
+    }
+    
+    convenience init(frame: CGRect ,with playlist: Playlist) {
+        self.init(frame: frame)
+        setUpDetailView(with: playlist)
+    }
+    
+    private let nibName = "TopDetailView"
+    private func commonInit() {
+        Bundle.main.loadNibNamed(nibName, owner: self, options: nil)
+        addSubview(contentView)
+    }
```

Comment:
```suggestion
    private func commonInit() {
        Bundle.main.loadNibNamed("TopDetailView", owner: self, options: nil)
        addSubview(contentView)
    }
```
để vào trong luôn cũng đc